chore(*-apps): use newest argoconfig

eyenx opened the pull request:
<!--
    Thank you for your contribution. Please use this template to help guide
    you towards preparing a PR that fullfills our merge requirements.
-->
# Description

makes our `*-apps` use the newest argoconfig version


<!-- Describe the changes your PR introduces here. -->

# Issues

Fixes #291
<!--
    Link related issues here, it's ok if this is empty but we do recommend that
    you create issues before working on PRs, issues on internal trackers are
    fine and need not be linked here.
-->

# Checklist

<!--
    Take care of the default items before marking your PR as ready for review,
    be prepared to add more items.
-->

* [x] I updated the version in Chart.yaml
* [x] I updated applicable README.md files using  `pre-commit run -a`
* [x] I documented any high-level concepts I'm introducing in `docs/`
* [x] If I updated a dependency tool, or app, this PR contains a short summary of the changes I'm pulling
* [x] CI is currently green and this is ready for review
* [x] I am ready to test changes after they are applied and released

<!--
    Please open PRs as Draft while you make CI green and/or finalise
    documentation. Your PR will be assigned to a CODEOWNER once you mark it
    as "Ready for Review".

   Once it is approved we will squash your changes onto the default branch
   and our trusty bot account will release them to the repository.
-->
Signed-off-by: Toni Tauro <toni.tauro@adfinis.com>

hairmare commented on charts/sentry-apps/Chart.yaml:
```diff
@@ -14,4 +14,4 @@ maintainers:
   - name: argoconfig
     repository: https://charts.adfinis.com
-    version: ~0.6.1
+    version: 0.7.4
```
i'm kinda not sure if this one doesn't introduce any bc breakage, will have to take a closer look

tongpu replied:
I'm not sure if sentry should still be a priority for us. I'm pushing our developers towards the cloud hosted solution right now.

eyenx replied:
we could keep this as is, und upgrade the  rest of the chart deps. 

Btw is it a patchfix bump or minor bump?

eyenx commented:
* did a minor bump of the chart version
* removed security-apps as it was already fixed by @tongpu 
* removed sentry-apps as we are unsure if it would create breaking changes

hairmare reviewed (approved):
LGTM since there are no more 0.6 -> 0.7 bumps involved and i'm pretty sure everything in 0.7.x is safe
